catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfPreStoragePlugin.java

```diff
@@ -210,14 +211,23 @@ private ContentItem createDerivedImage(String id, String qualifier, BufferedImag
         return null;
     }
 
-    protected String buildDerivedImageTitle(String title, String qualifier) {
+    String buildDerivedImageTitle(String title, String qualifier) {
+        String derivedImageTitle;
         String rootFileName = FilenameUtils.getBaseName(title);
 
-        return (StringUtils.isNotBlank(rootFileName) &&
-                StringUtils.isNotBlank(rootFileName.replaceAll("[^A-Za-z0-9]", ""))) // any human readable content
-                ? String.format(DERIVED_IMAGE_FILENAME_PATTERN, qualifier,
-                        rootFileName.replaceAll(INVALID_FILENAME_CHARACTER_REGEX, ""), JPG)
-                : String.format("%s.%s", qualifier, JPG);
+        // title must contain some alphanumeric, human readable characters, or use default filename
+        if (StringUtils.isNotBlank(rootFileName)
+                && StringUtils.isNotBlank(rootFileName.replaceAll("[^A-Za-z0-9]", ""))) {
+            String strippedFilename = rootFileName.replaceAll(INVALID_FILENAME_CHARACTER_REGEX, "");
+            derivedImageTitle = String
+                    .format(DERIVED_IMAGE_FILENAME_PATTERN, qualifier, strippedFilename, JPG)
+                    .toLowerCase();
+
+        } else {
```

Comment:
If you return in the if block, you can just fall through and return the default formatted string here without the else.